config/nav-cluster.js

```diff
@@ -26,10 +26,18 @@ export function allTypes($store) {
     const groupName = attrs.group || 'core';
     const api = $store.getters['cluster/byId'](API_GROUP, groupName);
 
-    // Skip non-preferred versions
-    if ( api?.preferredVersion?.version ) {
+    // Skip non-preferred versions if preferred version available
+    if ( api?.preferredVersion?.version) {
       if ( api.preferredVersion.version !== attrs.version ) {
-        continue;
+        if (attrs.version) {
+          const preferred = schema.id.replace(attrs.version, api.preferredVersion.version);
+
+          if (Object.keys(schemas).includes(preferred)) {
+            continue;
+          }
```

Comment:
What case is this addressing?  Are there apigroups which have a preferred version, but some of the resources are only available in a non-preferred version of them?  A new type that appears only in v2 but v1 is still preferred?

IDs should be treated as opaque strings (that you don't parse or try to interpret), and just replacing one version string with another could break things (e.g. if the API group name contains "v1").  Schemas have a group, version, and kind that are the individual parts that make up the ID.

The `includes()` makes this a O(n^2) loop over schemas which is likely noticeably slow with a few hundred entries?  So there should be a first pass to determine which version should be shown for a given group+kind, and then a 2nd pass to add the right one to the output array.  That makes it more like O(2n).